SwiftUI/Edge Debug Helper/Data/Repositories/DatabaseRepository.swift

```diff
@@ -172,8 +172,9 @@ actor DatabaseRepository {
                 "websocketUrl": appConfig.websocketUrl,
                 "httpApiUrl": appConfig.httpApiUrl,
                 "httpApiKey": appConfig.httpApiKey,
-                "mode": appConfig.mode,
+                "mode": appConfig.mode.rawValue,
```

Comment:
have you tested this code to make sure it upgrades existing app configs saved in the database properly?  Since secrete key is new, I thought it should be nil-able (optional) not to break existing users.  Did you test regression to see if someone changes an existing online playground config to a new shared key if it still works or not?